core/server/apps/private-blogging/tests/middleware_spec.js

```diff
@@ -311,7 +311,7 @@ describe('Private Blogging', function () {
             };
         });
 
-        it ('sets an error when there is no password', function (done) {
+        it.skip ('sets an error when there is no password', function (done) {
```

Comment:
Ahh actually I would probably delete this test completely and use a functional test instead.